src/arrayMethodSort.js

```diff
@@ -3,9 +3,21 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const value = this[i];
+
+          this[i] = this[j];
+          this[j] = value;
+        }
```

Comment:
The swapping of elements can be simplified by using array destructuring: [this[i], this[j]] = [this[j], this[i]]; This avoids the need for a temporary variable and makes the code more concise.